standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/txn/TxnHandler.java

```diff
@@ -1508,13 +1522,14 @@ public void commitTxn(CommitTxnRequest rqst) throws NoSuchTxnException, TxnAbort
                   if (partitionName != null) {
                     resource.append('/').append(partitionName);
                   }
-                  String msg = "Aborting [" + JavaUtils.txnIdToString(txnid) + "," + commitId + "]" + " due to a write conflict on " + resource +
+                  String msg = "Aborting " + " due to a write conflict on " + resource +
```

Comment:
I would keep it as it is, would be easier to troubleshoot

Reply:
Kept it as the same. Done.